modules/tensor_mechanics/src/actions/TensorMechanicsAction.C

```diff
@@ -76,7 +76,9 @@ TensorMechanicsAction::TensorMechanicsAction(const InputParameters & params)
     _planar_formulation(getParam<MooseEnum>("planar_formulation").getEnum<PlanarFormulation>()),
     _out_of_plane_direction(
         getParam<MooseEnum>("out_of_plane_direction").getEnum<OutOfPlaneDirection>()),
-    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : "")
+    _base_name(isParamValid("base_name") ? getParam<std::string>("base_name") + "_" : ""),
+    _point1(isParamValid("point1") ? getParam<Point>("point1") : Point(0, 0, 0)),
```

Comment:
I don't think we should have default values for the points that define the axis of rotation. If you want a cylindrical stress component, you would need to specify these. Also, this is a good opportunity to change the names of these parameters to something more descriptive of what they are, because `point1` and `point2` don't mean much in the context of the full set of parameters used in this Action.

Maybe we could specify `cylindrical_axis_point1` and `cylindrical_axis_point2`. Alternatively, we could just have the user specify `cylindrical_axis`, and provide `x`, `y`, or `z` as an enum in this context. It's less general, but simpler. These models are almost always set up with the axis of rotation being one of those axes. Those could be translated to two points to pass in to the Material that uses them. If the user has the more general case, they could set up the Materials themselves. Thoughts, @dschwen, @sapitts?